fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/VersionIndexTest.java

```diff
@@ -1059,13 +1295,13 @@ public void maxEverVersionWithinTransaction(int testFormatVersion, boolean testS
         try (FDBRecordContext context = openContext(maxEverVersionHook)) {
             MySimpleRecord record3 = MySimpleRecord.newBuilder().setRecNo(1066L).build();
             MySimpleRecord record4 = MySimpleRecord.newBuilder().setRecNo(1415L).build();
-            recordStore.saveRecord(record3);
+            FDBRecordVersion version3 = recordStore.saveRecord(record3).getVersion();
+            assertNotNull(version3);
             FDBRecordVersion version4 = getBiggerVersion(expectedMaxVersion);
-            recordStore.saveRecord(record4, version4);
+            FDBStoredRecord<?> storedRecord4 = recordStore.saveRecord(record4, version4);
 
             context.commit();
             assertNotNull(context.getVersionStamp());
-            FDBRecordVersion version3 = FDBRecordVersion.complete(context.getVersionStamp(), context.getLocalVersion(Tuple.from(record3.getRecNo())).get());
```

Comment:
Wait, this is wrong

Reply:
A previous version of this had some confusion as to which versions were complete and which were incomplete. The newer version I think doesn't have that any more.